ouroboros-consensus-cardano/src/unstable-byronspec/Ouroboros/Consensus/ByronSpec/Ledger/Mempool.hs

```diff
@@ -37,6 +39,11 @@ newtype instance Validated (GenTx ByronSpecBlock) = ValidatedByronSpecGenTx {
 
 type instance ApplyTxErr ByronSpecBlock = ByronSpecGenTxErr
 
+-- | This data family instance is not used anywhere but still required by the
+-- instance of @LedgerSupportsMempool ByronSpecBlock@
+newtype instance TxId (GenTx ByronSpecBlock) = TxId Int
+  deriving newtype NoThunks
```

Comment:
Good thing I actually specified that it was "not used but needed" such that you could find it and ask. Otherwise it would probably have gone unnnoticed.